storage/print/print.go

```diff
@@ -36,3 +37,9 @@ func (s *Storage) WriteAPICall(ctx context.Context, l *ctlog.Log, apiCall *apica
 	glog.Infof("%s: %s", l.Name, apiCall.String())
 	return nil
 }
+
+// WriteAPICall simply prints the STH and errors passed to it.
+func (s *Storage) WriteSTH(ctx context.Context, l *ctlog.Log, sth *ct.SignedTreeHead, errs []error) error {
+	glog.Infof("%s:\n\tSTH: %s\n\tVerification errors: %s", l.Name, sth, errs)
```

Comment:
Have you tested this? I don't think you can use `%s` with non-string arguments: https://play.golang.org/p/EXQj1IIUCYC

Reply:
Hmm.. so you're right.  That being said...

I did 'test' it, by running it and making it have errors... it worked... I don't know why, but it printed square brackets ith each error inside stringified... no idea why that worked! 

------------ After some testing ---------------

It looks like if you print a slice of things that are strings (https://play.golang.org/p/3jpz5yPyvHv) or have .String() methods (https://play.golang.org/p/GVQswBOrBhs) then it will Do The Right Thing... And errors use their Error() method for that too?  Or something?

If I change it to %v it seems to do the same, I can use that instead if you'd prefer?

Reply:
No it's fine if it works :)